lib/src/supabase_stream_builder.dart

```diff
@@ -40,11 +40,16 @@ class SupabaseStreamBuilder {
   /// `eq` filter used for both postgrest and realtime
   late final StreamPostgrestFilter? _streamFilter;
 
+  /// Used to identify which row has changed
+  final String _primaryKey;
```

Comment:
I know that primary key will be only one column 98% of the time, but there might be a case where there are multiple primary keys in a table. What do you think about making `_primaryKey` of type `List<String>` to support such cases?

```suggestion
  final List<String> _primaryKey;
```

Reply:
Are you sure? I know it's just stackoverflow, but it sounds plausible: https://dba.stackexchange.com/a/65290

Reply:
Okay as stated in the official doc, it's possible to create a key across multiple columns. But then all columns need to be in sum identical. Will add that.